blocks/common/button/button.css

```diff
@@ -0,0 +1,25 @@
+.button
+{
+    font-family: 'Raleway';

```

Comment:
It's a good practice to always have a fallback font in case the one you specified didn't load or didn't show yet.